src/quest_manager/views.py

```diff
@@ -51,6 +51,18 @@ class CategoryList(NonPublicOnlyViewMixin, LoginRequiredMixin, ListView):
     model = Category
 
 
+@method_decorator(staff_member_required, name='dispatch')
+class CategoryDetail(NonPublicOnlyViewMixin, DetailView):
+    model = Category
+
+    def get_context_data(self, **kwargs):
+
```

Comment:
I don't think you need this method.  `object` is already passed to the template by the default implementation of this method, so you can use `object` directly in your template with `object.title` and `object` instead of `c` (also, don't use uninformative variable names if you can help it)